ui/src/message_popup/renderers/question_interpreter.jsx

```diff
@@ -57,6 +59,28 @@ export default React.createClass({
     }
 
 
+    if (question.writeLong) {
+      return <LongTextResponse
+        key={key}
+        forceResponse={true}
+        responsePrompt="Notes:"
+        recordText="Next"
+        onLogMessage={onLogMessage}
+        onResponseSubmitted={onResponseSubmitted}
+      />;
+    }
+
+
+    if (question.timed) {
+      return <MinimalTimedView
```

Comment:
This is just the "response" or "interaction" bit, could we rename it to something that indicates that, maybe `TimedAutoAdvanceResponse`?